blog/draft/hoe-haven-jou-helpt.md

```diff
@@ -0,0 +1,117 @@
+---
+authors: [tom-ootes]
+tags: [haven, kubernetes, common-ground]
+draft: true
+---
+# Met Haven(+) word je écht digitaal autonoom, en kan je uiteindelijk zonder Amerikaanse cloudproviders
+
+Door alle perikelen die zich op het moment voltrekken in de Verenigde Staten staat het thema *Digitale Autonomie* ineens overal boven aan de agenda in overheids-it-land. Er wordt veel over gesproken, maar over hoe we precies onafhankelijk gaan worden lijkt nog geen consensus te bestaan.
```

Comment:
Welke "perikelen" in de VS? Ik zou spreken over "ontwikkelingen in de wereld", of dit gewoon helemaal weglaten en zeggen dat Digitale Autonomie hoog op het lijstje van de EU staat oid.

Reply:
Dit was enigszins komisch bedoeld, omdat er wmb vreselijke dingen gebeuren. Ik bagatelliseerde het wat door het perikelen te noemen. Ik zal het wel wat algemener schetsen inderdaad, dan snapt iedereen het.